packages/pwa-kit-create-app/assets/templates/@salesforce/retail-react-app/config/default.js.hbs

```diff
@@ -8,6 +8,33 @@
 const sites = require('./sites.js')
 module.exports = {
     app: {
+        // Commerce shopping agent configuration for embedded messaging service
+        // This enables an agentic shopping experience in the application
+        // This property is set to a serialized JSON string that contains the configuration for the commerce agent.
+        // The JSON string is set to the value of the COMMERCE_AGENT_SETTINGS environment variable.
+        // If the COMMERCE_AGENT_SETTINGS environment variable is not set, the feature is disabled.
+        commerceAgent:
+            process.env.COMMERCE_AGENT_SETTINGS ||
+            JSON.stringify({
```

Comment:
@avinashkasipathy why are we calling `JSON.stringify()`  as opposed to just having this as a plain javascript object?